ipatests/test_webui/test_loginscreen.py

```diff
@@ -401,3 +401,17 @@ def test_login_view(self):
         self.wait(0.5)
         self.check_elements_of_form(loginscreen.LOGIN_FORM)
         self.check_alerts(loginscreen.LOGIN_FORM)
+
+    @screenshot
+    def test_root_login(self):
+        """
+        Verify logging as root, using admin password, if there is
```

Comment:
I would put it as 
`Verify logging as root, using admin password. Root should be recognized as an alias for admin user`